capture.py

```diff
@@ -83,12 +90,10 @@ def get_sensor_values(i2c):
         break
     return reply, readings, ack, i, pd0, frame_counter
 
-
 class Node:
     def __init__(self):
         """
         Initialize the node.
-
```

Comment:
Why did you remove this line?

Reply:
I added it again, I didn't see it during merge